client/src/client.rs

```diff
@@ -898,12 +897,14 @@ pub trait RpcApi: Sized {
             opt_into_json(replaceable)?,
             opt_into_json(confirmation_target)?,
             opt_into_json(estimate_mode)?,
+            opt_into_json(avoid_reuse)?,
+            opt_into_json(fee_rate)?,
         ];
         self.call(
             "sendtoaddress",
             handle_defaults(
                 &mut args,
-                &["".into(), "".into(), false.into(), false.into(), 6.into(), null()],
+                &["".into(), "".into(), false.into(), false.into(), null(), null(), null(), null()],
```

Comment:
I guess `6.into()` is the confirmation target, is there a reason that was changed?

Reply:
According to the docs, the node uses the default value of `wallet -txconfirmtarget`, this is something each node operator will set on its own, when you are passing 6 as the default, you are overriding that configuration. I am passing `null()` here so that we don't send any value with the RPC command and the node therefore uses whatever it is configured as default internally to the node, without us having to tell it.

Reply:
If this is the case this change should probably be done as a separate patch (at the start of the PR) with full justification. This means next time someone is debugging they can use `git blame` on this line to see the commit that last touched this line and then check that commit to see why we no longer pass in 6.

Sorry for such a delayed response, I was not focused on this crate for along time.